virtualization/windowscontainers/manage-containers/container-storage.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Windows Server Container Storage
+description: How Windows Server Containers can use host & other storage types
+keywords: containers, volume, storage, mount, bindmount
+author: patricklang
+---
+
+
+
+# Overview
+
+<!-- Great diagram would be great! -->
+
+
+## Layer Storage
+
+This is all the files that are built into the container. Every time you `docker pull` then `docker run` that container - they are the same.
+
+
+### Where layers are stored and how to change it
+
+The layers are stored in `c:\programdata\docker` and split across the image and windowsfilter folders. Don't modify what's in these folders - they're carefully managed using commands such as:
+
+- [docker images](https://docs.docker.com/engine/reference/commandline/images/)
+- [docker rmi](https://docs.docker.com/engine/reference/commandline/rmi/)
+- [docker pull](https://docs.docker.com/engine/reference/commandline/pull/)
+- [docker load](https://docs.docker.com/engine/reference/commandline/load/)
+- [docker save](https://docs.docker.com/engine/reference/commandline/save/)
+
+If you want to use a path other than `c:\programdata\docker`, then you can change the `docker-root` configuration. Search for it on the [Docker Engine on Windows](docker/configure_docker_daemon.md) for an example of how to use it.
+
+
+### What works and doesn't
+
+Running containers can use most NTFS operations with the exception of transactions. This includes setting ACLs, and all ACLs are checked inside the container. If you want to run processes as multiple users inside a container, you can create users in your `Dockerfile` with `RUN net user /create ...`, set file ACLs, then configure later processes to run with that user with the [Dockerfile USER directive](https://docs.docker.com/engine/reference/builder/#user).
```

Comment:
You may want to clarify that this section still only applies to the layer files and not persistent volumes. I was initially confused.